tools/clang/unittests/HLSL/CompilerTest.cpp

```diff
@@ -1450,6 +1498,7 @@ void CompilerTest::TestPdbUtils(bool bSlim, bool bSourceInDebugModule, bool bStr
       /*HasVersion*/ false,
       /*IsFullPDB*/  true,
       /*hasHasAndPDBName*/false,
+      /*TestReflection*/false,
```

Comment:
Why false in this case?  Perhaps comment could help.

Reply:
Added a comment.